source/framework/test/files/metadata.rml

```diff
@@ -0,0 +1 @@
+<metadata  p1="75"  p2="12.32"  p3="Aloha" />
```

Comment:
It does not keep the convention that the xml header is the name of the c++ metadata class, which is `metadataTestClass`.